resources/views/gallery/_image.blade.php

```diff
@@ -24,9 +24,15 @@
                 <x-icon class="fa-regular fa-file text-4xl"></x-icon>
             </div>
         @else
-        <a class="block avatar grow relative cover-background"
-             style="background-image: url('{{ Img::crop(192, 144)->url($image->path) }}')">
-        </a>
+            @if ($image->focus_x && $image->focus_y)
```

Comment:
This should be a function on the Model that takes care of providing the image url with the focus or not